x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -25,14 +25,18 @@ import {
 import { DEFAULT_MAX_SIGNALS, DEFAULT_SEARCH_AFTER_PAGE_SIZE } from '../../../../common/constants';
 import type { CreateSecurityRuleTypeWrapper } from './types';
 import { getListClient } from './utils/get_list_client';
-import type { NotificationRuleTypeParams } from '../notifications/schedule_notification_actions';
-import { scheduleNotificationActions } from '../notifications/schedule_notification_actions';
-import { getNotificationResultsLink } from '../notifications/utils';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Can we remove these `eslint-disable`s?

Reply:
If you remove it, ESLint will show an error here. This stuff is either marked as deprecated or depends on stuff that is marked as deprecated, and it's all about rule legacy actions. We either need to get rid of the legacy actions implementation, or remove the deprecations. cc @elastic/security-solution-platform